tools/generation/test/fixtures/normal/info-multiline-folding.template

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+name: First template name
+path: normal/info-multiline-folding-
+es6id: 1.2.3
+info: |
+    This is an "info" field with an
+
+    empty line
+---*/
```

Comment:
This is a good idea, but I disagree with the expected behavior. I think that trailing whitespace should be preserved in all cases. I want to keep the test generator as transparent as possible, so that the output can be clearly traced back to the input. Let me know if you have a different interpretation.

Barring that, I'll update this patch to preserve trailing white space in the frontmatter in cases where it is present in the source material. I'll include the tests you've mentioned, of course.